apps/site/src/components/service-message-list.tsx

```diff
@@ -44,12 +44,12 @@ const ServiceMessageData: FunctionComponent<{
     return (
       <>
         <MdxCode>object</MdxCode>
-        <MdxTable>
+        <MdxTable width="100%">
           <thead>
             <tr>
-              <th>property</th>
+              <th style={{ width: "150px" }}>property</th>
```

Comment:
Bit magical/hacky but prevents this ugliness

![Screenshot 2022-10-17 at 19 39 16](https://user-images.githubusercontent.com/37743469/196256684-2f7e8747-b237-4b9e-a8d6-c0aa10a43b1f.png)